src/runtime/job.cpp

```diff
@@ -829,9 +829,17 @@ static void launch(JobTable *jobtable) {
     jobtable->imp->pipes[stdout_stream[0]] = entry;
     jobtable->imp->pipes[stderr_stream[0]] = entry;
     clock_gettime(CLOCK_REALTIME, &entry->job->start);
+    std::string stdin_file_str = "/dev/null";
+    int stdin_stream[2];
```

Comment:
I'm not going to block, but I really think this should be explicit via a tuple in wakelang.

Reply:
What do you mean exactly?

Reply:
Mentioned in chat but something like 

```
data StdInOption =
  DevNull
  Interactive
  File (filename)


def job = 
  makeExecPlan ......
  | setPlanStdinOption DevNull
  | runJob
```